### DISCUSSION TITLE
Permission Problems with Cloudron - Reload Cache

### DISCUSSION THREAD

**Discussion by ramillionaire** [kimai/kimai#3119] [On-Premise: Installation & Updates] [answered]
Hey there,
could anyone tell me how i can solve the permission problem regarding the reloading cache under cloudron ?
After  i run the command "bin/console kimai:reload --env=prod" the site crashes That something is wrong etc.... and i know it is the permission denied problem but in Cloudron "terminal" i can't make changes because it tells me that it is read only!
current user:group are www-data

thanks for any help

**Comment by kevinpapst** [comment, marked as answer]
You should ask Cloudron or read their docs: https://docs.cloudron.io/apps/kimai/

**Reply by ramillionaire** [reply]
thx - i only had to make sudo -u www-data ...now it works
